features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -277,7 +277,7 @@ data class RumConfiguration internal constructor(
          * By default, the SDK uses a [TimeBasedInteractionIdentifier] with a threshold of 3000ms.
          * @param lastInteractionIdentifier the [LastInteractionIdentifier] to use.
          */
-        fun setLastInteractionIdentifier(lastInteractionIdentifier: LastInteractionIdentifier): Builder {
+        fun setLastInteractionIdentifier(lastInteractionIdentifier: LastInteractionIdentifier?): Builder {
```

Comment:
This is a change in the existing public API, not breaking though. I believe we now allow `null` to disable identifier? Was using no-op implementation instead of `null` considered?

We probably need to add an explanation what `null` value does in the doc.